### ISSUE TITLE
The `Compare files` option from the Explorer doesn't show file names anymore

### ISSUE THREAD

**Issue by cateyes99** [Microsoft/vscode#43457]
- VSCode Version: 1.20.0
- OS Version: Windows 10

Steps to Reproduce:

1. Select the 1st file _README.md_ to compare from the **Explorer** by right clicking on _README.md_ and then clicking on the `Select for Compare` option
2. Now select the 2nd file _default.ps1_ to compare with _README.md_ by right clicking on _default.ps1_. It pops up the context menu. You can see the option to compare with the 1st file is just `Compare with Selected`
![image](https://user-images.githubusercontent.com/19678455/36082860-2e218e8e-1012-11e8-8a98-504158fe011c.png)

3. With previous Version **1.19.3**, it's `Compare 'README.md' with 'default.ps1'`. This old way is better as it's more informative and clearer than the new change. For example, i do a file comparison, after a while when i want to do file comparison again, the old way will tell me clearly what is the base file I compared last time if i want to reuse it as base, but might not be sure which it is now. If we want to make some change in the way to display this option, I think the **better** way is to display as `Compare with 'README.md'` as it's shorter but carries enough information 😄 
![image](https://user-images.githubusercontent.com/19678455/36082631-190c524c-1010-11e8-8dba-9378ddc1992b.png)

Does this issue occur when all extensions are disabled?: **Yes**

**Comment by octref** [comment]
@isidorn Starting with you since you were doing multiple selection in file explorer.

**Comment by isidorn** [comment]
@cateyes99 you are correct, this changed. However in this milestone we have adopted command in the file explorer which on the upside allows us to do nice multiselection, grouping, contributed commands. On the downside it does not allow us to dynamically change the names of commands (the issue you are seeing).
More about this change can be found in our release notes.
Closing as designed, since commands do not support dynamic label change.
